pkg/glusterfs/api/types.go

```diff
@@ -330,6 +330,16 @@ func (volExpandReq VolumeExpandRequest) Validate() error {
 	)
 }
 
+type VolumeCloneRequest struct {
+	Name string `json:"name,omitempty"`
```

Comment:
We don't actually clone by name do we? Generally, I would have expected a heketi volume Id here.

Reply:
No, this is the name of the volume that is being created from the original. By default it is `vol_{UUID}`, but users can pass it an optional `--name=...` just like with a `heketi-cli volume create --name=...` command.

Reply:
Note that the `VolumeCloneRequest` is sent to the `/volume/{uuid}/clone` endpoint, where the `{uuid}` is the reference to the source/original volume to be cloned.

Reply:
Yup. Got it. Thanks for the clarification. Something like this may be useful as a comment.